Assets/MixedRealityToolkit/Extensions/CameraExtensions.cs

```diff
@@ -33,9 +33,31 @@ public static float GetHorizontalFieldOfViewDegrees(this Camera camera)
         /// <param name="camera">The camera to check the point against</param>
         public static bool IsInFOV(this Camera camera, Vector3 position)
         {
-            return MathUtilities.IsInFOV(position, camera.transform, 
```

Comment:
Is there any benefit to use continuing to provide the `MathUtilities.IsInFOV` method, if it is slower than usingWorldToViewportPoint? Perhaps we should mention in a comment in that it's more performance to use `WorldToViewportPoint` when possible

Reply:
@julenka sorry somehow missed this comment. I left the MathUtilities because you don't require a camera with that. I guess I could have tried to re-write the MathUtilities to calculate a projection matrix from the fov and properties and then do the multiplication...but didn't want to mess with that